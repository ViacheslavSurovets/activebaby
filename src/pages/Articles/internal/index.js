export { ArticlesList } from './ArticlesList';
export { Article } from './Article';
export { ArticleListItemComponent } from './ArticleListitemComponent';
