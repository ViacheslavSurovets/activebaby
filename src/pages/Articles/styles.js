import styled from 'styled-components';
import { commonContentWrapper } from '@commonStyles';


export const ArticlesContent = styled.section`
  ${commonContentWrapper};
`;
