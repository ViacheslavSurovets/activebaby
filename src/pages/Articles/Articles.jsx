import React from 'react';
import { Route } from 'react-router-dom';
import { ArticlesList, Article } from './internal';
import { ArticlesContent } from './styles';
import PropTypes from 'prop-types';

const Articles = ( { match } ) => (
  <ArticlesContent>
    <Route exact path={ `${ match.path }` } component={ ArticlesList } />
    <Route exact path={ `${ match.path }/:articleId` } component={ Article } />
  </ArticlesContent>
);

export default Articles;

Articles.propTypes = {
  match: PropTypes.object
};

