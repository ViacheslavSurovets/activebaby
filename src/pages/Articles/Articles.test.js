import React from 'react';
import { shallow } from 'enzyme';
import Articles from './Articles';


describe ( 'Articles', () => {
  const mockMatch = {
    path: 'string'
  };
  it ( 'Articles Snapshot', () => {
    expect ( shallow ( <Articles match={ mockMatch } /> ) ).toMatchSnapshot ();
  } );
} );
