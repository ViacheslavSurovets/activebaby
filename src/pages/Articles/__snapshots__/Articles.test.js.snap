// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Articles Articles Snapshot 1`] = `ShallowWrapper {}`;
